.gitignore

```diff
@@ -1,5 +1,6 @@
 # build directorys
 /build.*/
+/builds
```

Comment:
Leftover?

Reply:
nope. possible name for a symlink to right location

Reply:
What symlink? AFAICS nothing uses `/builds` and this patch gets rid of the need to symlink anything?

Reply:
when you moved the build-root outside, you may want to access it fast.
so a symlink 'builds' to the new location will be fine ;)